addons/website/static/src/snippets/s_carousel_intro/options.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
```

Comment:
This is not needed anymore since this [commit](https://github.com/odoo-dev/odoo/commit/5f2c505836002ac7851c29c28d612f721d467bc4) :wink: